### ISSUE TITLE
Front-facing site should include a link to this repo

### ISSUE THREAD

**Issue by meelijane** [drupalmel/drupalmel#15]
So people can more easily contribute to it.

**Comment by Decipher** [comment]
Absolutely. A 'Fork this site' github ribbon or comment in the footer, or a dedicated block would be great. PRs welcomed.

**Comment by asherry** [comment]
one of these? https://github.com/blog/273-github-ribbons
I think I like black or orange

**Comment by GROwen** [comment]
+1 for orange, fits the colour scheme of DrupalMel nicely.

**Comment by asherry** [comment]
is there any info somewhere on the plan for feature names? I see a core feature, so I'll put this in there but maybe there are plans for something like a drupalmel_share feature or drupalmel_community feature, or some crap like that.

**Comment by Decipher** [comment]
There shouldn't be multiple features, no need. One feature per site is generally enough unless you need one piece of functionality to be disableable. Maybe in the future, but for now, just treat it like D8, a single configuration system.
